File: Easy/lc448.java
import java.util.*;
import java.util.List;
import java.util.ArrayList;
import java.util.HashSet;
class Solution448 {
    public List<Integer> findDisappearedNumbers(int[] nums) {
        Set<Integer> number = new HashSet<>();

        for (int num : nums) {
            number.add(num);
        }

        List<Integer> result = new ArrayList<>();

        for (int i = 1; i <= nums.length; i++) {
            if (!number.contains(i)) {
                result.add(i);
            }
        }

        return result;
    }
    public static void main(String [] args){
        Solution448 s = new Solution448();
        int[] nums = {4,3,2,7,8,2,3,1};
        System.out.println(s.findDisappearedNumbers(nums));
    }
}

File: Easy/lc448.py
class Solution(object):
    def findDisappearedNumbers(self, nums):
        number = set(nums)
        l1 = []
        for i in range(1,len(nums)+1):
            if i not in number:
                l1.append(i)
        return l1 
s=Solution()
print(s.minimumAbsDifference([4,3,2,7,8,2,3,1]))